src/access/AccessControl.cpp

```diff
@@ -368,6 +397,84 @@ CHIP_ERROR AccessControl::Check(const SubjectDescriptor & subjectDescriptor, con
     return CHIP_ERROR_ACCESS_DENIED;
 }
 
+#if CHIP_ACCESS_CONTROL_DUMP_ENABLED
+CHIP_ERROR AccessControl::Dump(const Entry & entry)
+{
+    CHIP_ERROR err;
+
+    ChipLogProgress(DataManagement, "----- BEGIN ENTRY -----");
```

Comment:
This is only here for ad hoc use during development, in case the developer needs to see what an entry looks like. It's not normally used. I changed it to detail anyways.